app/models/requests/request.rb

```diff
@@ -268,11 +268,9 @@ def build_requestable_with_items
 
       def build_requestable_from_data
         return if doc[:holdings_1display].nil?
+        @mfhd ||= 'thesis' if thesis?
         if @mfhd
-          params = build_requestable_params(holding: { @mfhd.to_sym.to_s => holdings[@mfhd] }, location: locations[holdings[@mfhd]["location_code"]])
-          requestable_items [Requests::Requestable.new(params)]
```

Comment:
This was where the original error was coming from (missing `=`).  I realized that this branch was untested.